README.md

```diff
@@ -123,6 +123,10 @@ Please note that this chart is not compatible with Helm 3 at this time.
 
 You can alter the variables in `charts/gatekeeper/values.yaml` to customize your deployment. To regenerate the base template, run `make manifests`.
 
+#### Policy library
```

Comment:
let's preserve the /library folder path and create a readme there to point to the new gatekeeper-library repo.